frontend/src/lib/components/LemonInput/LemonInput.tsx

```diff
@@ -5,6 +5,10 @@ import clsx from 'clsx'
 import { LemonButton } from 'lib/components/LemonButton'
 import { IconClose } from 'lib/components/icons'
 
+export enum LemonInputWidths {
+    Search = 340,
```

Comment:
This is a nice change!

Guides folk to how to extend in future and has one place that can accrue smelliness and push for future improvement 🙌